src/Lean/Elab/Extra.lean

```diff
@@ -163,32 +164,30 @@ private partial def toTree (s : Syntax) : TermElabM Tree := do
 where
   go (s : Syntax) := do
     match s with
-    | `(binop% $f $lhs $rhs) => processBinOp (lazy := false) s .anonymous f lhs rhs
-    | `(binop_lazy% $f $lhs $rhs) => processBinOp (lazy := true) s .anonymous f lhs rhs
-    | `(unop% $f $arg) => processUnOp s .anonymous f arg
+    | `(binop% $f $lhs $rhs) => processBinOp (lazy := false) s f lhs rhs
+    | `(binop_lazy% $f $lhs $rhs) => processBinOp (lazy := true) s f lhs rhs
+    | `(unop% $f $arg) => processUnOp s f arg
     | `(($e)) =>
       if hasCDot e then
         processLeaf s
       else
         go e
     | _ =>
-       match (← liftMacroM <| expandMacroImpl? (← getEnv) s) with
-       | some (macroName, s?) =>
-         let s' ← liftMacroM <| liftExcept s?
-         match s' with
-         | `(binop% $f $lhs $rhs) => processBinOp (lazy := false) s macroName f lhs rhs
-         | `(binop_lazy% $f $lhs $rhs) => processBinOp (lazy := true) s macroName f lhs rhs
-         | `(unop% $f $arg) => processUnOp s .anonymous f arg
```

Comment:
Just wanted to make sure, this should have been `macroName` instead of `.anonymous` right (origin e6caee97ec0c36c2b02e61e102f07b512b4d5cab)? Not sure if there was a particular reason that we wanted to erase elaborator information in the case of a macro expanding to an `unop%`.

Reply:
Oh, yeah this is probably just a typo considering the lines 166-168 above, NVM!

Reply:
Yeah, I guess so too. I didn't even notice.